plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaRecordSet.java

```diff
@@ -298,46 +336,4 @@ public void close()
             kafkaConsumer.close();
         }
     }
-
-    public static FieldValueProvider headerMapValueProvider(MapType varcharMapType, Headers headers)
```

Comment:
Avoid moving methods in existing code because it makes the git line history difficult to use.
And if you do have to do it - keep the "rearrangement only" changes in a separate commit so that it's easier to follow the diff.

Reply:
I've left the moving of the method aside.
Thank you for pointing this out.